rollup/plugins/bundlePlugins.js

```diff
@@ -131,6 +131,9 @@ export function makeTerserPlugin() {
           '_meta',
           // Object we inject debug IDs into with bundler plugins
           '_sentryDebugIds',
+          // These are used by instrument.ts in utils for identifying HTML elements & events
+          '_sentryCaptured',
```

Comment:
good thing we have good integration tests, that captured this! We also forgot this for `_sentryCaptured` (I guess we have no integration test covering this), so I also added this here @lforst